README.template.md

```diff
@@ -53,7 +53,7 @@ Add our dependency your `pom.xml`
 
```

Comment:
kinda does look like there are a bunch of changes to the README that aren't in this template though?